packages/mailgun/src/models/v1/emails/request/helper.ts

```diff
@@ -0,0 +1,17 @@
+import FormData = require('form-data');
+
+export const appendFilteredPropertiesToFormData = (
+  obj: Record<string, any>,
+  prefix: string,
+  formData: FormData,
+) => {
+  for (const [key, value] of Object.entries(obj)) {
+    if (key.startsWith(prefix) && value != null) {
+      if (value instanceof Date) {
+        formData.append(key, value.toISOString());
```

Comment:
Mailgun servers won't expect this format
- mailgun: https://documentation.mailgun.com/docs/mailgun/api-reference/openapi-final/tag/Messages/#tag/Messages/operation/POST-v3--domain-name--messages!path=o:deliverytime&t=request
- javascript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

Reply:
Here we are in the free-form variables and a choice has to be made in case the SDK user inputs a Date object. I chose ISO8601 but you're right, let's be consistent with he rest of the API